README.md

```diff
@@ -91,13 +91,13 @@ In such case, the claim set is printed to stdout in JSON format:
 
 ```json
 {
-  "profile": "http://arm.com/psa/2.0.0",
-  "partition-id": 1,
-  "security-life-cycle": 12288,
-  "implementation-id": "UFFSU1RVVldQUVJTVFVWV1BRUlNUVVZXUFFSU1RVVlc=",
-  "boot-seed": "3q2+796tvu/erb7v3q2+796tvu/erb7v3q2+796tvu8=",
-  "hardware-version": "1234567890123",
-  "software-components": [
+  "eat-profile": "http://arm.com/psa/2.0.0",
+  "psa-client-id": 1,
+  "psa-security-lifecycle": 12288,
+  "psa-implementation-id": "UFFSU1RVVldQUVJTVFVWV1BRUlNUVVZXUFFSU1RVVlc=",
+  "psa-boot-seed": "3q2+796tvu/erb7v3q2+796tvu/erb7v3q2+796tvu8=",
+  "psa-hardware-version": "1234567890123",
+  "psa-software-components": [
```

Comment:
please check psatoken/testvectors/json/ (e.g., test-token-valid-full.json) for an example of valid syntax

Reply:
The file  psatoken/testvectors/json/ (e.g., test-token-valid-full.json) should not be used. Have corrected the names now!